py/torch_tensorrt/dynamo/backend/test/utils.py

```diff
@@ -13,6 +13,7 @@
 
 from torch._functorch.aot_autograd import aot_module_simplified, make_boxed_compiler
 
+DECIMALS_OF_AGREEMENT = 4
```

Comment:
Would this constant not be necessary/helpful in the `export` testing? As in - could it instead go in `py/torch_tensorrt/dynamo/utils.py`?

Reply:
This parameter is used by tests which are currently written for `backend`. I don't have equivalent tests yet for export. I will work on those once this is evened out. After, that we can migrate this argument accordingly so that both can use it from a common place.